core/spec/models/spree/line_item_spec.rb

```diff
@@ -4,6 +4,27 @@
   let(:order) { create :order_with_line_items, line_items_count: 1 }
   let(:line_item) { order.line_items.first }
 
+  describe 'initial_price' do
+    it 'is available' do
+      expect(line_item).to respond_to(:initial_price)
+    end
```

Comment:
unnecessary : ) The tests below would fail anyhow.

The BigDecimal one is also superfluous imo